bindata/network/ovn-kubernetes/ovnkube-relay.yaml

```diff
@@ -0,0 +1,171 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-sbdb-relay
+  namespace: openshift-ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      This daemonset launches the ovn-kubernetes OVN SBDB relay networking components.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ovnkube-sbdb-relay
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 10%
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        app: ovnkube-sbdb-relay
+        component: network
+        type: infra
+        openshift.io/component: network
+        kubernetes.io/os: "linux"
+    spec:
+      serviceAccountName: ovn-kubernetes-node
+      hostNetwork: true
```

Comment:
Also as we found out we want to use pod anti affinity to limit relays to one-per-node like so:

```
      affinity:
        podAntiAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
          - labelSelector:
              matchExpressions:
              - key: app
                operator: In
                values:
                - ovnkube-sbdb-relay
            topologyKey: "kubernetes.io/hostname"
```